default.xml

```diff
@@ -14,4 +14,11 @@
 	<project path="multiap/agent" name="prplMesh-agent" />
 	<project path="multiap/tools" name="prplMesh-tools" />
 
+	<!-- upstream nng -->
+	<remote name="nng" fetch="ssh://git@github.com/nanomsg" />
+	<project path="nng" remote="nng" name="nng" revision="90467583b7544b68483334070518e29b00ec6d81" />
+
+	<!-- upstream safelibc -->
+	<remote name="safeclib" fetch="ssh://git@github.com/rurban" />
```

Comment:
same here, change the remote name to rurban and move to the top of the manifest where the other remotes are defined